packages/next/src/server-build.ts

```diff
@@ -1890,6 +1890,17 @@ export async function serverBuild({
             __next_data_catchall: nextDataCatchallOutput,
           }
         : {}),
+      // When bots crawl a site, they may render the page after awhile (e.g. re-sync),
+      // and the sub-assets may not be available then. In this case, the link to
+      // static assets could be not found, and return a 404 HTML. This behavior can
+      // bait the bots as if they found 404 pages. In Next.js it is handled on the
+      // server to return a plain text "Not Found". However, as we handle the "_next/static/"
+      // routes in Vercel CLI, the Next.js behavior is overwritten. Therefore, create a
+      // ".txt" file with "Not Found" content and rewrite any not found static assets to it.
+      '_next/static/not-found.txt': new FileBlob({
```

Comment:
```suggestion
      [path.posix.join('./', entryDirectory, '_next/static/not-found.txt)]: new FileBlob({
```
This needs to include the entryDirectory like in the route's `dest`.